plugins/alignments/src/LinearPileupDisplay/SharedLinearPileupDisplayMixin.ts

```diff
@@ -230,9 +228,18 @@ export function SharedLinearPileupDisplayMixin(
       /**
        * #action
        */
-      setFilterBy(filter: Filter) {
+      setFilterBy(filter: IFilter) {
         self.filterBy = cast(filter)
       },
+
+      /**
+       * #action
+       */
+      setJexlFilters(filters: string[]) {
```

Comment:
technically this is only able to push new things to the jexlFilters array, and is not able to clear it. so it's more like an "addToJexlFilters". Can probably just say self.jexlFilters=filters though to fix, or self.jexlFilters=cast(filters) if typescript complains